src/material/core/density/_index.scss

```diff
@@ -1,11 +1,11 @@
-// Taken from mat-density with small modifications to not rely on the new Sass module system.
+// Taken from mat-density with small modifications to not rely on the new Sass module
+// system, and to support arbitrary properties in a density configuration.
```

Comment:
Is this something they're going to add support for? How is this going to work when we move to using the official mdc density package?

Reply:
Not sure. I asked for it in the MDC collab channel, but haven't gotten any reply on this one yet. My thinking is that it should be okay doing this change in our local mod since we don't know yet how we will eventually rely on their utilities anyway. Plus, it's easy to switch to the limited properties they provide if we need.

I feel strongly though that the current limited properties are not sufficient for cases where multiple individual parts of a component scale with density. Might create a feature request for tracking.